inst/tests/tests.Rraw

```diff
@@ -13315,4 +13315,3 @@ test(1980, names(data.table(x)), "x")
 ###################################
 
 options(oldOptions)  # set at top of this file
-setDTthreads(0) # this has not been altered directly before so not sure if we should really reset it here
```

Comment:
this has not been set anywhere in the script so there is no need to restore. Number of threads should be controlled here by env var, so CRAN can be handled nicely. Assuming `getDTthreads` will work properly.